src/js/data/emojis.json

```diff
@@ -0,0 +1,13402 @@
+[
+  {
+    "group": 0,
+    "hexcode": "1f600",
+    "label": "grinning face",
```

Comment:
Wonder if we should localize labels. As far as I know, emoji search is localized in other messaging apps or even in iOS emoji keyboard. Maybe we can add a `labelKey` key that represents a i18n string.

Reply:
One of the reasons to choose the [`emojibase`](https://www.npmjs.com/package/emojibase) dataset is that it provides localisation support with the data.
https://emojibase.dev/docs/datasets/#supported-locales

Currently, we use the `en` locale dataset. This can be changed depending on the user's device locale (just like i18n translation) post-MVP. 😄

Reply:
Awesome, thanks for the clarification 👍